autoload/vp4.vim

```diff
@@ -112,6 +112,30 @@ function! s:PerforceWrite(cmd)
     endif
     execute command
 endfunction
+
+" Function to get the path of the file
+function! s:ExpandPath(file)
+    if exists("g:base_path_replacements")
+        echom "We have a base path replacements"
+        let l:oldPath = expand('%:p')
+        let l:replacements = items(g:base_path_replacements)
```

Comment:
see `plugin/vp4.vim:30` for how options are set and assigned a default value.  Please prefix option names with `vp4_`